setup.py

```diff
@@ -38,7 +38,7 @@ def get_version(package):
     install_requires=[
         'Django==1.10.5',
         'dj-database-url==0.4.2',
-        'psycopg2cffi==2.7.5',
+        'psycopg2==2.7.3.1',
         'djangorestframework==3.5.3',
         'drf-extensions==0.3.1',
         'djangorestframework-composed-permissions==0.1',
```

Comment:
Not really sure why these are all pinned. They should maybe be unpinned and tested but I will leave that to the dev team 😊

Reply:
@JayH5 These are pinned and then updated by requires.io: https://github.com/praekelt/seed-auth-api/pull/50 . The reason being is that it's kind of end-user software, no other software is going to import and use this software.

Reply:
Right.. but nobody is clicking merge on those PRs..

Reply:
Yup, there hasn't been development on this in a LONG time. Somehow it has survived though, and not needed any maintenance.